config/README.md

```diff
@@ -1,18 +0,0 @@
-# config
```

Comment:
Instead of deleting this, can we instead explain what is now in this directory?

Reply:
Sure, I can add something quickly if you think it's valuable. I think this directory might go away when we refactor the repo into multiple modules though. Having multiple modules share a config directory seems unusual.

Reply:
Just a sentence would be fine. We can discuss removing the directory with the division.

Reply:
Will do.